docs/en/ingest-management/elastic-agent/debug-standalone-elastic-agent.asciidoc

```diff
@@ -0,0 +1,203 @@
+[[debug-standalone-agents]]
+= Debug standalone {agent}s
+
+When you run standalone {agent}s, you are responsible for monitoring the status
+of your deployed {agent}s. You cannot view the status or logs in {fleet}.
+
+Use the following tips to help identify potential issues.
+
+Also refer to <<fleet-troubleshooting>> for guidance on specific problems.
+
+NOTE: You might need to log in as a root user (or Administrator on Windows) to
+run these commands.
+
+[discrete]
+== Check the status of the running {agent}
+
+To check the status of the running {agent} daemon and other processes managed by
+{agent}, run the `status` command. For example:
+
+[source,shell]
+----
+elastic-agent status
+----
+
+Returns something like: 
+
+[source,yaml]
+----
+Status: HEALTHY
+Message: (no message)
+Applications:
+  * metricbeat  (HEALTHY)
+                Running
+  * filebeat    (STOPPING)
+                (no message)
+----
+
+By default, this command returns the status in human-readable format. Use the
+`--output` flag to change it to `json` or `yaml`.
+
+For more information about this command, refer to
+<<elastic-agent-status-command>>.
+
+//REVIEWERS: Looks like JSON and YAML return status codes instead of healthy,
+//stopping, etc. Where can I find the mapping of status code to status?
+
+[discrete]
+[[inspect-standalone-agent-logs]]
+== Inspect {agent} and related logs
+
+If the {agent} status is unhealthy, or behaving unexpectedly, inspect the logs
+of the running {agent}. The log location varies by platform. Refer to
+<<installation-layout>> for the correct location.
+
+//REVIEWERS: Can we add more guidance here about how to explore logs, what to
+//look for, etc.
```

Comment:
The `elastic-agent` logs would be in the folder specified by installation-layout, and beats/fleet-server would be in folders named by the output, i.e., `default`.
Endpoint is a special case as its logs are written in its install dir.

For debugging we generally start with looking for `error` level logs and investigate from there.
Repeated lines can show whatever strange behaviour is occurring (such as connection issues).
Sometimes there are no issues that can be found through the logs, and it may be that the agent or one of it's processes had an oom incident and the OS should be checked.